example/package.json

```diff
@@ -8,9 +8,10 @@
   },
   "dependencies": {
     "moment": "^2.17.1",
-    "react": "16.0.0-alpha.6",
-    "react-native": "0.44.0",
-    "react-native-pathjs-charts": "file:../",
+    "react": "^15.4.1",
+    "react-native": "^0.42.0",
+    "react-native-pathjs-charts": "file:///Users/luculent/Documents/Git/react-native-pathjs-charts",
```

Comment:
Hard-coded user directory won't work for anyone - please change back to `file:../`